scripts/ci/tools/freespace.py

```diff
@@ -0,0 +1,68 @@
+#
```

Comment:
You need a shebang as the first line so that python is used to execute the script.

 Right now the CI fails, because it thinks it is a bash script.

https://bash.cyberciti.biz/guide/Shebang

In our case we use:

```
#!/usr/bin/env python3
```